crates/tauri-plugin-holochain-service-consumer/android/src/main/java/DisconnectedNotice.kt

```diff
@@ -0,0 +1,135 @@
+package com.plugin.holochain_service_consumer
+
+import android.app.Activity
+import android.content.Intent
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.util.Log
+import android.widget.Button
+import android.widget.FrameLayout
+import android.widget.TextView
+import androidx.cardview.widget.CardView
+
+class DisconnectedNotice(private val activity: Activity, private val servicePackage: String) {
```

Comment:
class for managing the "disconnected" notice that is displayed when the consumer app cannot connect to the HolochainService